index.js

```diff
@@ -375,6 +379,29 @@ class GithubScm extends Scm {
     }
 
     /**
+     * Check validity of github webhook event signature
+     * @method _checkSignature
+     * @param   {String}    secret      The secret used to sign the payload
+     * @param   {String}    payload     The payload of the webhook event
+     * @param   {String}    signature   The signature of the webhook event
+     * @returns {boolean}
+     */
+    _checkSignature(secret, payload, signature) {
+        const hmac = crypto.createHmac('sha1', secret);
+
+        hmac.setEncoding('hex');
+        hmac.write(JSON.stringify(payload), 'utf-8');
```

Comment:
Just copy over this
https://github.com/mhazy/hapi-github-webhooks/blob/master/lib/index.js#L55-L56
I install this locally and the test it out with our actual signing secret, it works fine :)